### PR TITLE
2Update README.md

### LINKED ISSUE #363: TEst
TEst issue

### PR DESCRIPTION
**Pull request by smarroufin**
Resolves #315 
Fixes #363 
Fix #382 
Fix #381 
Resolves smarroufin/test2#32

aaaa2234565

### CODE REVIEW

**Comment by benjamincanac** [comment]
test

**Comment by benjamincanac** [comment]
test

**Comment by benjamincanac** [comment]
test

**Comment by benjamincanac** [comment]
test

File: README.md [hunk 1]
```diff
@@ -21,3 +21,4 @@ ftest
 fdsfds
 gfd
 fdsfds
+test
```
Comment (smarroufin) [inline README.md:24]: la

**Pull request closed without merging**